projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -418,7 +418,7 @@
 
 		&[type="radio"]::before {
 			border-radius: 50%;
-			transform: translateY(15%);
+			transform: translateY(0.15rem);
```

Comment:
Are these `transform` changes intended as part of this PR? It looks like they're part of fixing up the positioning of the radio button from #43531 instead 🤔

Reply:
Ah dang. Thanks for catching. I'll revert. I might have mistakenly used that branch as a base when checking out this one.

Reply:
Done in [507e24c](https://github.com/Automattic/jetpack/pull/43545/commits/507e24c91ef87d8c85a938630a718230f4f12e7b)

Reply:
Uh, that didn't seem to do anything..

Reply:
Until I refresh the page 😄 Ignore me